apps/3000-home/next.config.js

```diff
@@ -58,6 +58,13 @@ const nextConfig = {
     const { isServer } = options;
     // used for testing build output snapshots
 
+    config.resolve.alias['@module-federation/runtime'] = require.resolve(
```

Comment:
might not need this, but ensures it gets the built versions not the tspath resolve/transpile